includes/definitions/rittal-lcp.yaml

```diff
@@ -4,14 +4,16 @@ type: environment
 icon: rittal
 mib_dir:
     - rittal
-over:
-    - { graph: device_processor, text: 'Processor Usage' }
-    - { graph: device_ucd_memory, text: 'Memory Usage' }
-    - { graph: device_storage, text: 'Storage Usage' }
 discovery:
     -
         sysObjectId:
             - .1.3.6.1.4.1.2606.7
     - sysDescr:
         - 'Rittal LCP'
-        
\ No newline at end of file
+discovery_modules:
+    storage: 0
+    mempools: 0
+    processors: 0
+over:
```

Comment:
I removed this because the LCP is an appliance Cooling Unit and shows information about memory, storage and CPU with this enabled. We have no interaction with this information and we can't change anything on this side. So it's not a good information to show that, in my opinion. In my case on our two LCP the memory is saturated even when the unit is not running. So it's probably not important to show this information. 🤔